Added content about baseline's package. (#46)
pharo-open-documentation/pharo-wiki: General/Baselines.md

  package: 'BaselineOfMyProject'
``` 

> This class should be in a package separated from other package's projects. By convention, we name this package holding the baseline with the same name as the baseline. To summarize, `BaselineOfMyProject` class is in `BaselineOfMyProject` package.

Reviewer: It's not just a convention I think because if you have a different name Metacello will not be able to resolve it.

> This class should be in a package separated from other package's projects. By convention, we name this package holding the baseline with the same name as the baseline. To summarize, `BaselineOfMyProject` class is in `BaselineOfMyProject` package.

Reviewer: ```suggestion
This class should be in a package separated from other package's projects. This package holding the baseline should have the same name as the baseline. To summarize, `BaselineOfMyProject` class is in `BaselineOfMyProject` package.
```
